docs/developers/node-tutorial.md

```diff
@@ -0,0 +1,218 @@
+content with the following script:
+
+```sh
+#!/bin/sh
+
+set -o errexit -o nounset
+
+CHAINID="devnet-2"
+app="./celestia-appd"
+
+# Build genesis file incl account for passed address
+coins="1000000000000000uceles"
+$app init $CHAINID --chain-id $CHAINID 
+$app keys add validator --keyring-backend="test"
+$app keys add celes --keyring-backend="test"
+# this won't work because the some proto types are decalared twice and 
+# the logs output to stdout (dependency hell involving iavl)
+$app add-genesis-account $($app keys show validator -a --keyring-backend="test") $coins
+$app add-genesis-account $($app keys show celes -a --keyring-backend="test") $coins
+$app gentx validator 5000000000uceles --keyring-backend="test" --chain-id $CHAINID
+$app collect-gentxs
+
+# Set proper defaults and change ports
+sed 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.celestia-app/config/config.toml
+sed 's/timeout_commit = "5s"/timeout_commit = "1s"/g' ~/.celestia-app/config/config.toml
+sed 's/timeout_propose = "3s"/timeout_propose = "1s"/g' ~/.celestia-app/config/config.toml
+sed 's/index_all_keys = false/index_all_keys = true/g' ~/.celestia-app/config/config.toml
+
+# Start the celestia-app
+$app start --grpc.enable
+```
+
+Next, run the following from the root-directory of celestia-app:
+
+```sh
+cp build/celestia-appd contrib/
+cd contrib
+./single-node.sh
+```
+
+This will instantiate the single-node.sh script which will run a single-node
+network for celestia-app.
+
+## Celestia Node
+
+In another terminal window, run the following:
+
+```sh
+cd $HOME
+git clone https://github.com/celestiaorg/celestia-node.git
+cd celestia-node/
+make install
+```
+
+Verify its installed by running the following command:
+
+```sh
+$ celestia version
+Semantic version: v0.2.0
+Commit: 1fcf0c0bb5d5a4e18b51cf12440ce86a84cf7a72
+Build Date: Fri 04 Mar 2022 01:15:07 AM CET
+System version: amd64/linux
+Golang version: go1.17.5
+```
+
+Now, let's instantiate a celestia-light node:
+
+```sh
+./build/celestia light init
+```
+
+With the light-node installed, we can proceed to copying over the keys from
+the celestia-app to the celestia-node so they share the same key.
+
+```sh
+mkdir ~/.celestia-light/keys/keyring-test
+cp ~/.celestia-app/keyring-test/* ~/.celestia-light/keys/keyring-test
+```
+
+This will copy over the keys to your Celestia Light node.
+
+Let's now run the Celestia Light node with a GRPC connection to the running
+Celestia App instance:
+
+```sh
+./build/celestia light start --core.grpc 127.0.0.1:9090
+```
+
+## Node API Calls
+
+With both Celestia App and Celestia Node running in separate terminals, we can
+now open a new terminal session to start making API calls!
+
+First, let's find our wallet public addresses generated by
+the `single-node.sh` script.
+
+Run the following (Note: You don't have to use VIM to open the file, it is
+used here for demonstration)
+
+```sh
+vim ~/.celestia-app/config/genesis.json
+```
```

Comment:
Most online instructions in see about editing files use `nano`. Maybe use that instead? Ships by default with Ubuntu.